inst/shiny-server/server/upload.R

```diff
@@ -87,7 +87,10 @@ observeEvent(input$repository.load_button, {
   DataList$data <- c(DataList$data, data)
   update_menu_visibility(attr(DataList$data, 'suite'))
   # set_format_func(attr(DataList$data, 'suite'))
-  set_color_scheme("Default", get_algId(DataList$data))
+  algids <- get_algId(DataList$data)
+  if (!all(algids %in% get_color_scheme_dt()[['algnames']])) {
```

Comment:
In which scenario will this happen?

Reply:
This happens when you first load some data, change the colorscheme, and then upload more data (as done for example in Dominik's usecases)

Reply:
looks good to me